client/components/room/Video.jsx

```diff
@@ -295,16 +295,20 @@ class Video extends React.Component {
   toggleVideo() {
     if(localStream.getVideoTracks()[0].enabled) {
       localStream.getVideoTracks()[0].enabled = false;
+      document.getElementById('video').src = ('../img/video-off.png');
     } else {
       localStream.getVideoTracks()[0].enabled = true;
+      document.getElementById('video').src = ('../img/video-on.png');
     }
   }
 
   toggleAudio() {
     if(localStream.getAudioTracks()[0].enabled) {
       localStream.getAudioTracks()[0].enabled = false;
+      document.getElementById('audio').src = ('../img/audio-off.png');
```

Comment:
I love the new image buttons! I am wondering if we could set the source of these components using the the redux Store, so that react know to properly update the component when the buttons are clicked. With the deadline fast approaching though, I think we're good 😄